modules/ROOT/pages/subqueries/subqueries-in-transactions.adoc

```diff
@@ -591,6 +602,189 @@ RETURN n.num, s.errorMessage;
 REPORT STATUS can only be used when specifying ON ERROR CONTINUE or ON ERROR BREAK
 ----
 
+[role=label--new-5.21]
+[[concurrent-transactions]]
+== Concurrent transactions
+
+By default, `CALL { ... } IN TRANSACTIONS` is single-threaded; one CPU core is used to sequentially execute batches.
+
+However, `CALL` subqueries can also execute batches in parallel by appending `IN [n] CONCURRENT TRANSACTIONS`, where `n` is a concurrency value used to set the maximum number of transactions that can be executed in parallel.
+This allows `CALL` subqueries to utilize multiple CPU cores simultaneously, which can significantly reduce the time required to execute a large, outer transaction.
+
+[NOTE]
+The concurrency value is optional.
+If not specified, a default value based on the amount of available CPU cores will be chosen. 
```

Comment:
More strictly it is "based on the number of logical processors available to the Java Virtual Machine". But I am not sure if we need to be that specific here? It is similar to the documentation for the setting `server.cypher.parallel.worker_limit`.

Right now we also support a negative value in the same way as with that setting. We should probably document that as well to be transparent. Maybe something like:
"If specified as a negative number, the concurrency will be the number of logical processors available on the Java virtual machine reduced by the absolute value of that number."
or
"If specified as a negative number, the concurrency will be the number of available CPU cores reduced by the absolute value of that number."